client/packages/coldchain/src/Equipment/DetailView/DetailView.tsx

```diff
@@ -72,13 +75,21 @@ export const EquipmentDetailView = () => {
       value: 'Summary',
     },
     {
-      Component: <StatusLogs />,
+      Component: draft === undefined ? null : <StatusLogs assetId={draft.id} />,
```

Comment:
to ensure that the assetId is provided - just makes it easier to handle in the list component